test/pantry-test.js

```diff
@@ -58,6 +58,13 @@ describe('Pantry', () => {
     let isTrue = pantry.compareIngredients(pantrySupply[0], greenHam.requiredIngredients[0]);
     expect(isTrue).to.equal(true); 
   });
+  
+  it('can compareIngredient IDs whether or not they\'re assigned to ' +
+  '.id or .ingredient', () => {
+    let isTrue = pantry.compareIngredients(greenHam.requiredIngredients[0], pantrySupply[0]);
+    expect(isTrue).to.equal(true); 
```

Comment:
I'd say this is evaluating to true because both arguments are evaluating to undefined.